### DISCUSSION TITLE
Regex in shorturl/longurl

### DISCUSSION THREAD

**Discussion by jamesmr89** [YOURLS/YOURLS#3426] [General]
Trying to replace an in house tool that does this:

https://domain/shorturl/(\d+) and map it to https://mysite/mypage.aspx?param=$1

This would translate something like this from short url to long:

https://domain/shorturl/12345 and map it to https://mysite/mypage.aspx?param=12345

Could this be accomplished through a plugin?

Thanks

**Comment by ozh** [comment]
Virtually everything can be done with a plugin. See https://docs.yourls.org/guide/extend/possible-with-a-plugin.html
